.github/workflows/test.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
-      - run: npm ci
-      - run: npm run lint
-      - run: npm run format
-      - run: npm test
+      - run: yarn install --immutable
```

Comment:
I think within gh action, the `CI` env variable should be set, so `yarn install` should already been run in `--immutable` mode